src/api/services/login-service.ts

```diff
@@ -179,6 +180,14 @@ export class LoginService extends CoreWebService {
         }).map(response => {
             this.setLogInUser(response.entity);
             this.dotcmsConfig.setUser(response.entity);
+
+            // TODO: we need to refactor the appconfig because it's not updating after login/logout
+            this.appConfigurationService.getConfigProperties().subscribe(res => {
+                let langCode = res.dotcmsConfig.configParams.user.languageId.split('_')[0];
```

Comment:
Note: this should be returned in a more semantic way from the BE, so FE should just do user.language.code, instead of splitting the strings